ydb/docs/en/core/public-talks/_includes/publications/2023/medium.md

```diff
@@ -0,0 +1,29 @@
+### Migrations in YDB using “goose” {#2023-pub-medium-goose}
+
+![Migrations in YDB using goose](./_includes/goose.webp ={{pub-cover-size}})
+
+Any production process that works with a database will require a schema migration sooner or later. The migration updates the database’s table structure from one version to the next. Schema migrations can be done manually by executing an `ALTER TABLE` query or by using specialized tools. One such tool is called goose. In this [article](https://blog.ydb.tech/migrations-in-ydb-using-goose-58137bc5c303) we see how goose provides schema management in a project and has supported YDB (a distributed open-source database) since v3.16.0.
```

Comment:
```suggestion
Any production process that works with a database will require a schema migration sooner or later. The migration updates the database’s table structure from one version to the next. Schema migrations can be done manually by executing an `ALTER TABLE` query or by using specialized tools. One such tool is called "goose". In this [article](https://blog.ydb.tech/migrations-in-ydb-using-goose-58137bc5c303), we see how goose provides schema management in a project and has supported YDB (a distributed open-source database) since v3.16.0.
```